sources/packages/backend/apps/api/src/route-controllers/application-offering-change-request/_tests_/e2e/application-offering-change-request.institutions.controller.getCompletedApplications.e2e-spec.ts

```diff
@@ -153,6 +153,8 @@ describe("ApplicationOfferingChangeRequestInstitutionsController(e2e)-getComplet
             id: declinedBySABCApplicationOfferingChange.id,
             applicationNumber:
               applicationWithDeclinedBySABCApplicationOfferingChange.applicationNumber,
+            applicationId:
```

Comment:
is these fields required for institution endpoints?

Reply:
OK as discussed.

Reply:
@andrepestana-aot, The discussion was to remove it from completed right? @dheepak-aot @guru-aot @sh16011993

Reply:
Yes, removed from completed and in-progress.